week1/armstrong.rb

```diff
@@ -9,5 +9,16 @@
 # and `false` otherwise.
 
 def armstrong_number(n)
-    raise NotImplementedError # Delete this line and start coding here
+
+    l = n.digits
+    sum = 0
+    for i in l do
+        sum += i**3
+    end
+    
+    if sum == n
+        return true
+        
+    else
+        return false
```

Comment:
```suggestion
    if sum == n
        return true
    else
        return false
    end
```
Syntax error -> `if` block should have an ending `end`